apps/02RockPaperScissors.js

```diff
@@ -9,9 +9,46 @@ var rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
```

Comment:
I like this, as long as you can remember that this change was made. I prefer having it attached to every variable so I can always see it.